### DISCUSSION TITLE
这样做感觉超过了法律底线……

### DISCUSSION THREAD

**Discussion by LoveBodhi** [v2fly/v2ray-core#913] [General]
所以我不能接受，官网也无法访问。

**Comment by Cwek** [comment]
然后呢？或者在工作单位和家庭网络之间建立一条个人用加密的通道来管理家里的服务器呢？

工具是死的，人是活的。你不能接受，出门左转，谢谢。

**Comment by LoveBodhi** [comment]
然后就可以喝了（笑），使用外国代理访问中国大陆无法访问的网站才算超过法律底线。

**Comment by kulongwangzhi85** [comment]
菜刀在家里可以切菜，拿到外面还可以砍人！工具怎么用还得看你自己。
